uwblueprint/ccbc
# Table display titles don’t match the actual review

## Notion ticket link

[Table display titles don’t match the actual review](https://www.notion.so/uwblueprintexecs/Table-display-titles-don-t-match-the-actual-review-e6c90a9a8b834ce5b0261a697aba9a3f)



## Implementation description
* Instead of passing in rowIndex as parameter, use reviewId instead to search for the index



## Steps to test
1. Sort columns and verify edit, preview, and delete actions work on the correct review
2. Filter reviews and verify edit, preview, and delete actions work on the correct review



## What should reviewers focus on?
* Code correctness & testing


## Checklist
